.github/workflows/job_tensorflow_hub_models_tests.yml

```diff
@@ -108,12 +108,28 @@ jobs:
           python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers-*
 
       - name: Install TF Hub tests requirements
-        run: python3 -m pip install -r ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_hub_tests/requirements.txt
+        run: python3 -m pip install -r ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_tests/requirements.txt
 
       - name: TensorFlow Hub Tests - TF FE
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_hub_tests/ -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_tests/ -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v --scope tf_hub
+        env:
+          TYPE: ${{ inputs.event == 'schedule' && 'nightly' || 'precommit'}}
+          TEST_DEVICE: CPU
+
+      - name: TensorFlow Hugging Face Tests - TF FE
+        run: |
+          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_tests/test_tf_convert_model.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v --scope hf
+        env:
+          TYPE: ${{ inputs.event == 'schedule' && 'nightly' || 'precommit'}}
+          TEST_DEVICE: CPU
+
+      - name: TensorFlow Other Tests (OMZ) - TF FE
+        run: |
+          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
```

Comment:
We need to think how to deal with it. To have both nightly validations for TFHUB and HF in the single validation is not possible because time limit is 5h and this is not enough.
Let us sync offline to think

Reply:
As agreed, made Hugging Face models launch for precommit only.